org.kde.kclock.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "org.kde.kclock",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kclock",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=xdg-data/sounds",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.Solid.PowerManagement",
+        "--own-name=org.kde.kclockd"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kirigamiaddons",
+            "config-opts": [ "-DBUILD_TESTING=OFF" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://invent.kde.org/libraries/kirigami-addons.git", "branch": "release/21.05", "commit": "4724edc523c855bba54c34366c272e41ad022869" } ]
```

Comment:
As mentionner, on kweather,  having branch and the commit will fail the build as soon as the branch is updated. Please omit the branch.

Also if you actually have a tag, it's better.

Reply:
Generally you're encouraged to use the download.kde.org links and tarballs, not least since @flathub/kde have some tooling to make sure that these all get updated with new KDE releases.

Reply:
kirigami-addons doesn't have a tarball at the moment, so I can update it once it does.

Reply:
There is a new kirigami-addons tarball now: https://download.kde.org/unstable/kirigami-addons/0.2/